sorc/chgres_cube.fd/search_util.f90

```diff
@@ -15,7 +15,7 @@ module search_util
 
  contains
 
-!> @brief Replace undefined surface values.
+!! @brief Replace undefined surface values.
```

Comment:
Take out the brief tag for functions/subroutines. It's only needed for file-level documentation blocks.

Reply:
Done